robot.py

```diff
@@ -4,41 +4,45 @@
 from robotpy_toolkit_7407.sensors.limelight import Limelight
 from oi.OI import OI
 from wpilib import SmartDashboard
+import command
+from robot_systems import Robot
 
-
-class Robot(wpilib.TimedRobot):
+class _Robot(wpilib.TimedRobot):
     def __init__(self):
         super().__init__()
 
     def robotInit(self):
         # Initialize Operator Interface
         OI.init()
         OI.map_controls()
-        period = .03
-        commands2.CommandScheduler.getInstance().setPeriod(period)
+        # period = .03
```

Comment:
```suggestion
        period = .03
```
Don't comment out the period setters, they are used for ensuring that our code runs with a consistent loop time.